src/main/java/com/pio/studentManagement/utils/SaveStudentResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.pio.studentManagement.utils;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SaveStudentResponse {
+    private boolean success;
```

Comment:
add an enum here
```suggestion
    private Status status;
```

`public enum Status {
    SUCCESS,
    ALREADY_EXISTS,
    VALIDATION_ERROR
}
`

add above Enum in response and try to return it from service layer.

Reply:
added an enum with SUCCESS and ALREADY_EXIST and used accordingly